### ISSUE TITLE
Php and html indentation breaks when using html5 plugin

### ISSUE THREAD

**Issue by alx741** [jwalton512/vim-blade#42]
When editing a blade file the *filetype* becomes `BLADE` but then auto indentation for **HTML** and **PHP** (within `<?php`, `?>` tags no longer works as spected).

Should blade indent bring in html and php indent files?

For proper html autoindent inside php files i'm using this plugin:

                      Plug 'captbaritone/better-indent-support-for-php-with-html'

**Comment by jwalton512** [comment]
@alx741 thanks for the insight.

I do seem to experience the same issue with indenting between `<?php` and `?>`

I don't experience any issue, however, with indentation for html.

**Comment by alx741** [comment]
So far i notice it only preserves the indentation of the previous tag, but doesn't trigger the proper indentation levels for nested tags.

Thank your for this plugin btw, its very useful!.

**Comment by alx741** [comment]
Apparently vim-blade doesn't play nice with the html5 plugin, when i remove the last one, the html autoindent starts working ok in blade. I have no clue why though.

https://github.com/othree/html5.vim

**Comment by jwalton512** [comment]
@alx741 thanks for the clarity. Will try to research both items.

**Comment by moccalotto** [comment]
I too get the same error as @alx741

**Comment by nivv** [comment]
:+1:

**Comment by nivv** [comment]
I currently do this when re-indenting the whole blade file:

```vim
"Indent blade, <D-ﬁ> == CMD+L | Set filetype to html, then indent, then switch back to blade 
autocmd FileType blade noremap <buffer> <D-ﬁ> :set filetype=html<cr> gg=G'' :set filetype=blade <cr>
```

It works fine, it won't indent `@foreach` as such, it will look like below instead, which I'm fine with. Now my only annoyance is that when you insert a new line it isn't indented automatically.

**Comment by adriaanzon** [comment]
I'm currently trying to fix the php indentation. I came across [this conditional](https://github.com/jwalton512/vim-blade/blob/53622d538717d8bf15b2efefeeff2ed51d7ec0bd/indent/blade.vim#L44-L46) while I was at it.

It has a syntax error (one `=` instead of two) so vim ignores it. When I 'fix' the syntax error, the indentation gets all messed up.

Does anyone know why it's there? Do we need it for something or should it be removed?

**Comment by jwalton512** [comment]
resolved with #51
